<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleAnswerQuestion</class>
 <widget class="QDialog" name="MultipleAnswerQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="QModeLbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Persian" country="Iran"/>
   </property>
   <property name="text">
    <string>نوع  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="QTitleLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>64</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Persian" country="Iran"/>
   </property>
   <property name="text">
    <string>موضوع  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="QMode">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Persian" country="Iran"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Persian" country="Iran"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Timer">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 4);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ثبت پاسخ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skipBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>رد کردن</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="questionText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>461</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="locale">
    <locale language="Persian" country="Iran"/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="answer1RB">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="answer2RB">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="answer3RB">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="answer4RB">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
